spec/Confirmation_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'Confirmation'
+
+
+describe Confirmation do
+  subject(:confirmation) { described_class.new("+447880521694") }
```

Comment:
if you had injected the client, you could have replaced it with a double here to avoid sending real sms during testing.